src/table/internal.tsx

```diff
@@ -222,7 +222,7 @@ const InternalTable = React.forwardRef(
       onFocusMove: moveFocus,
       onResizeFinish(newWidth) {
         const widthsDetail = columnDefinitions.map(
-          (column, index) => newWidth[getColumnKey(column, index)] || (column.width as number) || DEFAULT_COLUMN_WIDTH
+          (column, index) => newWidth[getColumnKey(column, index)] || (column.width as number)
```

Comment:
The type of widths array in the event detail is ReadonlyArray<number>. The current line produces ReadonlyArray<number | undefined> it looks like.

Reply:
And you are correct. When changing it to the correct type, I am getting a lot of TS errors and this is why I am cancelling this PR as it is a breaking change